packer/scripts/windows/winrm_max_memory.ps1

```diff
@@ -5,6 +5,9 @@
 # this file be licensed under the Apache-2.0 license or a
 # compatible open source license.
 
+echo "The max amount of the winrm memory must be 2GB smaller than max host memory, no less"
```

Comment:
A little confused, so for example, the winrm memory has to be set more than 126 if the max is 128? or it has to be exact 126.

Reply:
I think it is minimum to be 126G.